kotsadm/kustomize/overlays/dev/deployment.yaml

```diff
@@ -42,3 +42,7 @@ spec:
               value: http://replicated-app:3000
             - name: API_ADVERTISE_ENDPOINT
               value: http://127.0.0.1:30065
+            # - name: STORAGE_BASEURI
+            #   value: "docker://kotsadm-storage-registry:5000"
+            # - name: STORAGE_BASEURI_PLAINHTTP
+            #   value: "true"
```

Comment:
should these stay commented out?

Reply:
I was torn on this one. Enabling this makes the environment use the new OCI storage. But there's no migration, so if these are present, it will break envs. 

Any thoughts on how you'd like to see this rolled out as an alpha feature to the skaffold environment?